test/client/config.js

```diff
@@ -1,25 +1,133 @@
 /* @flow */
 
-import { getPayPalDomain, getPayPalAPIDomain, getPayPalLoggerDomain, buildPayPalUrl, buildPayPalAPIUrl, getPayPalLoggerUrl } from '../../src';
+import { insertMockSDKScript, getPayPalDomain, getPayPalAPIDomain, getPayPalLoggerDomain, buildPayPalUrl, buildPayPalAPIUrl, getPayPalLoggerUrl } from '../../src';
+
+beforeEach(() => {
+    window.__ENV__ = 'test';
+});
 
 describe(`config cases`, () => {
 
-    it('should successfully get the paypal domain', () => {
+    it('should successfully get the testing paypal domain', () => {
         const expectedPayPalDomain = 'mock://www.paypal.com';
 
         if (getPayPalDomain() !== expectedPayPalDomain) {
             throw new Error(`Expected paypal domain to be ${ expectedPayPalDomain }, got ${ getPayPalDomain() }`);
         }
     });
 
-    it('should successfully get the paypal api domain', () => {
+    it('should successfully get the production paypal domain', () => {
+        window.__ENV__ = 'production';
+        const expectedPayPalDomain = 'https://www.paypal.com';
+
+        if (getPayPalDomain() !== expectedPayPalDomain) {
+            throw new Error(`Expected paypal domain to be ${ expectedPayPalDomain }, got ${ getPayPalDomain() }`);
+        }
+    });
+
+    it('should successfully get the sandbox paypal domain', () => {
+        const expectedPayPalDomain = 'https://www.sandbox.paypal.com';
+        window.__ENV__ = 'sandbox';
+
+        insertMockSDKScript({
+            query: {
+                'client-id': 'sb'
+            }
+        });
+
+        if (getPayPalDomain() !== expectedPayPalDomain) {
+            throw new Error(`Expected paypal domain to be ${ expectedPayPalDomain }, got ${ getPayPalDomain() }`);
+        }
+    });
+
+    it('should successfully get the sandbox v2 paypal domain', () => {
+        const expectedPayPalDomain = 'https://www-v2.sandbox.paypal.com';
+        window.__ENV__ = 'sandbox';
+
+        insertMockSDKScript({
+            query: {
+                'client-id': 'sb',
+                'env':         'sandbox_v2'
```

Comment:
Nit: this spacing seems a little off. I think `sb` and `sandbox_v2` should align. Maybe its the github ui

Reply:
I was my `lint --fix ` fault, fixed